### DISCUSSION TITLE
Is it possible to integrate html 5 projects?

### DISCUSSION THREAD

**Discussion by Raj2032** [tauri-apps/tauri#3121] [General]
If I wanted to integrate something like [`html5-qrcode`](https://github.com/mebjas/html5-qrcode) into my web project, would tauri support importing third party projects such as this?

This third party project does qr code scanning in web pages.

![image](https://user-images.githubusercontent.com/29293830/147424279-f75d8d8a-74ae-4742-92dd-4875e7350e5e.png)

**Comment by amrbashir** [comment]
Yes, if a library works in a normal web browser, it will work with Tauri.

**Reply by Joe23232** [reply]
I can integrate the two together though?

**Reply by amrbashir** [reply]
yeah

**Reply by Raj2032** [reply]
I see thanks mate
